src/core/Header/HeaderButton.tsx

```diff
@@ -69,6 +86,7 @@ export const HeaderButton = (props: HeaderButtonProps) => {
       },
       className,
     ),
+    ['aria-label']: isActive ? buttonTranslations.page : undefined,
```

Comment:
This should really be `aria-current` and it does not need translations. Also, use `location` not `page`. If the user wants `page` for any reason, they can still override.

See https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaCurrent

Reply:
Fixed

Reply:
Because I confused computed attribute names, and linter does not care about this particular thing apparently. Fixed.